packages/create-react-native-library/src/index.ts

```diff
@@ -787,6 +790,41 @@ async function create(argv: yargs.Arguments<any>) {
     }
   }
 
+  // Some of the passed args can already be derived from the generated package.json file.
+  const ignoredArgs: ArgName[] = [
+    'slug',
+    'description',
+    'author-name',
+    'author-email',
+    'author-url',
+    'repo-url',
+    'example',
+    'react-native-version',
+  ];
+  const ignoredAnswers = ignoredArgs.map((argName) => questions[argName]?.name);
+  const standardArgs = ['_', '$0', 'name'];
+
+  const filteredAnswers = Object.fromEntries(
```

Comment:
Maybe some name like `libraryMetadata` to make it more obvious what it is?

```suggestion
  const libraryMetadata = Object.fromEntries(
```